addons/web/static/src/search/search_bar/search_bar.js

```diff
@@ -70,16 +74,12 @@ export class SearchBar extends Component {
         const subItems = "subItems" in options ? options.subItems : this.subItems;
 
         const tasks = [];
-        const expandedSearchItems = this.searchItems.filter((item) => expanded.includes(item.id));
-        for (const searchItem of expandedSearchItems) {
-            // expand a properties should lazily fetch the definition, and generate
-            // searchItems, so the normal field logic is used for properties
-            const field = this.fields[searchItem.fieldName];
-
-            if (field.type === "properties" && searchItem.type === "field") {
-                tasks.push({ id: searchItem.id, prom: this.insertPropertiesSearchItems(searchItem, field, query) });
-            } else if (!subItems[searchItem.id]) {
-                tasks.push({ id: searchItem.id, prom: this.computeSubItems(searchItem, query) });
+        for (const id of expanded) {
+            const searchItem = this.getSearchItem(id);
+            if (searchItem.type === "field" && searchItem.fieldType === "properties") {
+                tasks.push({ id, prom: this.getSearchItemsProperties(searchItem) });
```

Comment:
is it still search items ? (method is called `getSearchItemsProperties` and not `getSubItemsProperties`)